### ISSUE TITLE
Failure to mount drive on M1 Mac/BigSur

### ISSUE THREAD

**Issue by silkfirefly** [joncampbell123/dosbox-x#2601]
I downloaded the dosbox-x version which is meant to support 64 bit play. After a few runs in Terminal was able to open the app (described in someone else's issue with Mac), but now I am unable to mount any directory other than Z. I've gone through every walkthrough/simplification I can find, but the end result is always "Failed to mount the PhysFS drive." What is the workaround here? I have my games in a folder inside Applications, should it be a different location? Not super tech savvy, just trying to get back some of those childhood feel good games!

**Comment by Wengier** [comment]
What if you try something like the following:

`MOUNT C .`

`MOUNT D ~`

`MOUNT E /`

And see if they work for you?

**Comment by silkfirefly** [comment]
> What if you try something like the following:
> 
> `MOUNT C .`
> 
> `MOUNT D ~`
> 
> `MOUNT E /`
> 
> And see if they work for you?

Hello! Those did not work for me. However, as I said I am not tech savvy. I realized the drop down menu allowed me to choose drives to mount, and then it allowed me to select which folder I wanted. So, I picked C: for nostalgia's sake, then the already created game folder, and voila! So thank you all the same, as somehow I ended up figuring out how to properly use the program 🤦🏻‍♀️
